nextmv/nextmv/cloud/manifest.py

```diff
@@ -629,9 +633,21 @@ class ManifestOptions(BaseModel):
 
     An option is a parameter that configures the decision model.
     """
+    format: Optional[list[str]] = None
+    """A list of strings that define how options are transformed into command line arguments.
+
+    Use `{{name}}` to refer to the option name and `{{value}}` to refer to the option value.
+    For example, `["-{{name}}", "{{value}}"]` will transform an option named `max_vehicles`
+    with a value of `10` into the command line argument `-max_vehicles 10`.
+    """
 
     @classmethod
-    def from_options(cls, options: Options, validation: OptionsEnforcement = None) -> "ManifestOptions":
+    def from_options(
+        cls,
+        options: Options,
+        validation: OptionsEnforcement = None,
+        format: Optional[list[str]] = None,
+        ) -> "ManifestOptions":
```

Comment:
Indenting seems off here. I think unindent to the left.

Reply:
FYI @dirkschumacher : we use ruff as the linter. 
Also, I missed a change in `options.py`. That should indeed not be necessary. :relaxed:

Reply:
I think we are confusing the custom formatting of the options part here that is necessary for apps not adhering to our convention. Our own options adhere to our convention though, so, not necessary to let them deal with anything. At least, not in the current scope. 
We always need to carefully distinguish `nextmv` and `nextmv.cloud` namespaces here. The latter is applicable to non-python apps, etc.

Reply:
Ok, so in my mind I don't necessarily try to distinguish `nextmv` and `nextmv.cloud` because the idea (at least for me) is that they always play nicely with each other and not drift apart (too much, at least). Ok, so with what you say:

> necessary for apps not adhering to our convention

Then it makes sense, for as you put it: apps built with the SDK whould always follow convention. Thank you for clarifying.